src/WebJobs.Script/Description/Rpc/WorkerFunctionDescriptorProvider.cs

```diff
@@ -40,5 +43,27 @@ protected override IFunctionInvoker CreateFunctionInvoker(string scriptFilePath,
             });
             return new WorkerLanguageInvoker(Host, triggerMetadata, functionMetadata, inputBindings, outputBindings, inputBuffer);
         }
+
+        protected override Collection<ParameterDescriptor> GetFunctionParameters(IFunctionInvoker functionInvoker, FunctionMetadata functionMetadata,
+            BindingMetadata triggerMetadata, Collection<CustomAttributeBuilder> methodAttributes, Collection<FunctionBinding> inputBindings, Collection<FunctionBinding> outputBindings)
+        {
+            var parameters = base.GetFunctionParameters(functionInvoker, functionMetadata, triggerMetadata, methodAttributes, inputBindings, outputBindings);
+
+            var bindings = inputBindings.Union(outputBindings);
+
+            var returnsSomething = bindings.SingleOrDefault(b => b.Metadata.ReturnsSomething);
```

Comment:
It would be good to handle the exception that will result from having a configuration issue and more than one binding setup to handle the return value (a descriptive message that will clearly identify the issue). As is, this will likely result in a pretty cryptic error.